aemedge/blocks/article-tags/article-tags.css

```diff
@@ -21,6 +21,6 @@ main .article-tags div.tag-list a {
 }
 
 main .article-tags div.tag-list a:hover {
-    background-color: var(--udexColorNeutral);
-    border: 1px solid var(--udexColorGrey2);
-}
\ No newline at end of file
+    background-color: var(--udexColorGrey3);
```

Comment:
Previous variable didn't seem to exist, so I replaced it with hover value for grey tag from Frontify.